app/helpers/global_activities_helper.rb

```diff
@@ -31,7 +33,7 @@ def generate_link(message_item, activity)
 
     case obj
     when User
-      return popover_for_user_name(obj, team, false, true)
+      return "[@#{obj.full_name}~#{obj.id.base62_encode}]"
```

Comment:
I'm slightly triggered :)

Reply:
What's wrong here?

Reply:
Sorry, didn't want to 'request changes'. Just 'base62' triggers me a bit :)

Reply:
Nothing wrong.